### DISCUSSION TITLE
model of the wind turbine?

### DISCUSSION THREAD

**Discussion by Lunahannen** [PaddlePaddle/PaddleSpatial#11] [BaiduKDDCup2022] [answered]
Dear organizers,
Can you tell us about the model of the wind turbine?

Thank you for your time.

**Comment by Lunahannen** [comment]
Is DFIG？

**Comment by zhoujingbo** [comment, marked as answer]
Hi, after a discussion, we think this information should not be disclosed.

Thanks a lot.

Regards
